Source/gui/SNTBinaryMessageWindowController.m

```diff
@@ -92,6 +92,8 @@ - (void)loadWindow {
     NSString *eventDetailText = [[SNTConfigurator configurator] eventDetailText];
     if (eventDetailText) {
       [self.openEventButton setTitle:eventDetailText];
+      [self.openEventButton setKeyEquivalent:@"\r"];
+      [self.openEventButton setKeyEquivalentModifierMask:!NSEventModifierFlagCommand];
```

Comment:
We'd prefer if this _did_ require Command to be held. On the off-chance that a block dialog appears in the middle of someone typing, we don't want them to inadvertently dismiss the dialog without being able to see it. Cmd+Enter makes this significantly less likely.

Reply:
Sounds good, I can incorporate that shortly! thanks for reviewing